deepmd/pt/train/training.py

```diff
@@ -484,7 +484,11 @@ def collect_single_finetune_params(
                             if i != "_extra_state" and f".{_model_key}." in i
                         ]
                         for item_key in target_keys:
-                            if _new_fitting and ".fitting_net." in item_key:
+                            if _new_fitting and (
+                                (".fitting_net." in item_key)
+                                or (".out_bias" in item_key)
+                                or (".out_std" in item_key)
```

Comment:
I think it should be `".descriptor" not in item_key`. When the model has other variables in the future, is it expected to keep it or replace it?

Reply:
In current version, the two ways of writing `".descriptor" not in item_key` and `(".fitting_net." in item_key) or (".out_bias" in item_key) or (".out_std" in item_key)` are fully equivalent. Please @iProzd  take a look at this.

Reply:
I think `".descriptor." not in item_key` is better. Yet @Chengqian-Zhang please check and ensure that they are equivalent.

Reply:
<img width="628" alt="image" src="https://github.com/user-attachments/assets/9e32e0ea-e36d-42b7-abc6-364b860c196b">

Reply:
I an sure that in current version, if "descriptor" and "fitting" do not in target_keys, only "out_bias" and "out_std" remain.